app/database/migration/server/index.ts

```diff
@@ -22,4 +22,15 @@ export default schemaMigrations({migrations: [
             }),
         ],
     },
+    {
+        toVersion: 3,
```

Comment:
```suggestion
        toVersion: 2,
```

no need to bump as the previous migration has not been shipped